poc_mobile_verifier/src/subnetwork_rewards.rs

```diff
@@ -106,6 +113,52 @@ mod test {
         }
     }
 
+    fn bytes_per_s(mbps: i64) -> i64 {
+        mbps * 125000
+    }
+
+    fn cell_type_weight(cbsd_id: &String) -> Decimal {
+        CellType::from_cbsd_id(cbsd_id)
+            .expect("unable to get cell_type")
```

Comment:
do you have to use expect here, the consensus has been to avoid (outside of tests ) as it will result in a panic

Reply:
This is a function only in the test sub-module so it should have the same effect